app/web/features/auth/login/LoginForm.tsx

```diff
@@ -87,63 +74,59 @@ export default function LoginForm() {
 
   return (
     <>
-      {sent && (
-        <TextBody className={authClasses.feedbackMessage}>
-          {t("auth:login_page.form.no_password_login_prompt")}
-        </TextBody>
-      )}
       <form className={authClasses.form} onSubmit={onSubmit}>
         <InputLabel className={authClasses.formLabel} htmlFor="username">
           {t("auth:login_page.form.username_field_label")}
         </InputLabel>
         <TextField
           className={authClasses.formField}
-          disabled={sent}
           fullWidth
           id="username"
           inputRef={register({ required: true })}
           name="username"
           variant="standard"
         />
-        {!loginWithLink && (
-          <>
-            <InputLabel className={authClasses.formLabel} htmlFor="password">
-              {t("auth:login_page.form.password_field_label")}
-            </InputLabel>
-            <TextField
-              className={authClasses.formField}
-              fullWidth
-              id="password"
-              name="password"
-              inputRef={(inputElement) => {
-                if (inputElement) {
-                  inputElement.focus();
-                }
-                register(inputElement, { required: true });
-              }}
-              type="password"
-              variant="standard"
-            />
-          </>
-        )}
+        <InputLabel className={authClasses.formLabel} htmlFor="password">
+          {t("auth:login_page.form.password_field_label")}
+        </InputLabel>
+        <TextField
+          className={authClasses.formField}
+          fullWidth
+          id="password"
+          name="password"
+          inputRef={register({ required: true })}
+          type="password"
+          variant="standard"
+        />
         <div className={classes.loginOptions}>
-          <FormControlLabel
-            className={classes.rememberSwitch}
-            control={<Switch size="small" />}
-            label={t("auth:login_page.form.remember_me")}
+          <Controller
+            control={control}
+            name="rememberDevice"
+            defaultValue={true}
+            render={({ onChange, value }) => (
+              <FormControlLabel
+                className={classes.rememberSwitch}
+                value="rememberDevice"
```

Comment:
```suggestion
```
Would this still work if it's left out? It's a bit confusing to have a value of `rememberDevice`when it's supposed to be the field's name

Reply:
Let's see!